src/pretix/presale/views/event.py

```diff
@@ -104,6 +108,29 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class EventIcalDownload(EventViewMixin, View):
+    def get(self, request, *args, **kwargs):
+        if not self.request.event:
+            raise Http404(_('Unknown event code or not authorized to access this event.'))
+
+        cal = Calendar()
+        cal.add('version', '2.0')
+        cal.add('prodid', '-//{}//{}//'.format(self.request.event.organizer.name, self.request.event.name))
```

Comment:
Sorry for replying late... I was quite busy recently. 
Regarding `prodid`, I included LANGUAGE_CODE as the last parameter. What do you think?
```
cal.add('prodid', '-//pretix//{}//{}'.format(settings.PRETIX_INSTANCE_NAME, 
                                                                    settings.LANGUAGE_CODE.upper()))
```